### DISCUSSION TITLE
[Feature Request] Ability to take notes with timestamp

### DISCUSSION THREAD

**Discussion by guleriaakshit** [PaulWoitaschek/Voice#2123] [Ideas]
I want app to have ability to takes notes with timestamps.

Warm regards,
Uzi

**Comment by PaulWoitaschek** [comment]
Isn't that what bookmarks already are? (Press the bookmark icon in the playback screen and you can make notes with timestamps (aka bookmarks)

**Reply by guleriaakshit** [reply]
Thanks

**Comment by AmritasyaPutra** [comment]
I would like the ability to edit the timestamp later, it would be simple to add this ability in the edit dialog.

features/bookmarks/dialogs/EditBookmarkDialog.kt
